research/object_detection/model_lib.py

```diff
@@ -649,6 +649,15 @@ def create_estimator_and_inputs(run_config,
       predict_input_fn=predict_input_fn,
       train_steps=train_steps)
 
+def _compare_fn(best_eval_result, current_eval_result):
+  # export the model with the best mAP@0.5IOU
+  default_key = "DetectionBoxes_Precision/mAP@.50IOU"
```

Comment:
Since you are hardcoding the key here, I guess it only works with coco detection metric? If so, please document that.